connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+    },
+    {
+      "id": "operationDetails",
+      "label": "Operation Details"
+    },
+    {
+      "id": "output",
+      "label": "Output Mapping"
+    },
+    {
+      "id": "errors",
+      "label": "Error Handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:google-sheets:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "id": "authenticationType",
+      "label": "Type",
+      "group": "authentication",
+      "type": "Dropdown",
+      "value": "bearer",
+      "choices": [
+        {
+          "name": "Bearer Token",
+          "value": "bearer"
+        },
+        {
+          "name": "Refresh Token",
+          "value": "refresh"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.authType",
+        "key": "authentication.authType"
+      }
+    },
+    {
+      "label": "Bearer Token",
+      "description": "Enter Google API Bearer token that is authorized to interact with desired Drive",
+      "group": "authentication",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.bearerToken"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "bearer"
+      }
+    },
+    {
+      "label": "Client Id",
+      "description": "Enter Google API Client Id",
+      "group": "authentication",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.oauthClientId"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "refresh"
+      }
+    },
+    {
+      "label": "Client Secret",
+      "description": "Enter Google API Client Secret",
+      "group": "authentication",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.oauthClientSecret"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "refresh"
+      }
+    },
+    {
+      "label": "Refresh Token",
+      "description": "Enter Google API Refresh token that is authorized to interact with desired Drive",
+      "group": "authentication",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.oauthRefreshToken"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "refresh"
+      }
+    },
+    {
+      "id": "operationType",
+      "group": "operation",
+      "type": "Dropdown",
+      "value": "createSpreadsheet",
+      "choices": [
+        {
+          "name": "Create spreadsheet",
```

Comment:
please use the title case for all properties:
Instead of "Create spreadsheet" use "Create Spreadsheet"